templates/pages/management/ticketsatisfaction.html.twig

```diff
@@ -0,0 +1,64 @@
+{% import "components/form/fields_macros.html.twig" as fields %}
+
+{% block more_fields %}
+    <div class='alert alert-success d-flex align-items-center' role='alert'>
+        <span class='ms-2'>
+        <i class='fas fa-xl fa-info-circle'> </i> &nbsp;
+            {{ __("After 12 hours, you can no longer modify your notice.") }}
+        </span>
+    </div>
```

Comment:
Need to be added with ```fields.htmlField``` (to control width)

![image](https://github.com/glpi-project/glpi/assets/7335054/c7d72163-b537-4386-9b39-556e7a9cdd47)

Green border should only on left side (see Lock tab from Computer)

![image](https://github.com/glpi-project/glpi/assets/7335054/f2b3abeb-212e-4505-b31b-8c219a6fc91a)